release.go

```diff
@@ -215,36 +216,35 @@ func main() {
 		if err != nil {
 			log.Fatal(err)
 		}
-		err = update.CopyLatest(*promoteReleasesBucketName, *promoteReleasesPlatform)
+		err = update.CopyLatest(*promoteReleasesBucketName, *promoteReleasesPlatform, false)
```

Comment:
For easy reading and changing later this promoteRelasesCmd case could have
`const dryRun bool = false`